ionic2/app/pages/about/about.page.scss

```diff
@@ -1,3 +1,7 @@
 .about {
 
 }
+
+ion-item a{
+  text-decoration: none;
```

Comment:
All style sheets are manually imported into `/app/theme/app.core.scss` and thus apply to the entire dom. With webpack I've seen a really cool way of importing css files directly so you could do something like this:

```(javascript)
import * as aboutStyle from './about.component.scss';

@Page({
    styles: [aboutStyle]
})
```

Unfortunately, we are not using webpack but browserify and I don't know if it supports this as well (maybe through plugins?).